src/Nest/XPack/Info/XPackInfo/XPackInfoResponse.cs

```diff
@@ -0,0 +1,87 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using Newtonsoft.Json;
+
+namespace Nest
+{
+	public interface IXPackInfoResponse : IResponse
+	{
+		[JsonProperty("build")]
+		XPackBuildInformation Build { get; }
+		[JsonProperty("license")]
+		MinimalLicenseInformation License { get; }
+		[JsonProperty("features")]
+		XPackFeatures Features { get; }
+		[JsonProperty("tagline")]
+		string Tagline { get; }
+	}
+
+	public class XPackInfoResponse : ResponseBase, IXPackInfoResponse
+	{
+		public XPackBuildInformation Build { get; internal set; }
+		public MinimalLicenseInformation License { get; internal set; }
+		public XPackFeatures Features { get; internal set; }
+		public string Tagline { get; internal set; }
+	}
+
+	public class XPackBuildInformation
+	{
+		[JsonProperty("date")]
+		public DateTime Date { get; internal set; }
+		[JsonProperty("hash")]
+		public string Hash { get; internal set; }
+	}
+
+	public class MinimalLicenseInformation
+	{
+		[JsonProperty("uid")]
+		public string UID { get; internal set; }
+
+		[JsonProperty("type")]
+		public LicenseType Type { get; internal set; }
+
+		[JsonProperty("mode")]
+		public LicenseType Mode { get; internal set; }
+
+		[JsonProperty("status")]
+		public LicenseStatus Status { get; internal set; }
+
+		[JsonProperty("expiry_date_in_millis")]
+		public long ExpiryDateInMilliseconds { get; set; }
```

Comment:
I'm going to leave this as, might be useful for `elasticsearch` itself to start returning `date`.